src/screens/About/About.tsx

```diff
@@ -52,75 +58,41 @@ const About = ({ children }: { children: React.ReactNode }) => {
       <Header>
         <Content>
           <Typography variant="h3" component="h1">
-            About Covid Act Now
+            {content.pageHeader}
           </Typography>
         </Content>
       </Header>
       <Content>
         <StapledSidebar sidebar={sidebar}>
           <SectionHeader variant="h4" component="h4" id="can">
-            Covid Act Now
+            {content.introHeader}
           </SectionHeader>
-
-          <Typography variant="body1" component="p">
-            <a
-              target="_blank"
-              rel="noopener noreferrer"
-              href="https://blog.covidactnow.org/covid-modeling-values/"
-            >
-              Guided by common values
-            </a>
-            , Covid Act Now is a multidisciplinary team of technologists,
-            epidemiologists, health experts, and public policy leaders working
-            to provide disease intelligence and data analysis on COVID in the
-            U.S.
-          </Typography>
-          <Typography variant="body1" component="p">
-            We published the first version of our model on March 20. Over 10
-            million Americans have used the model since. We’ve engaged with
-            dozens of federal, state, and local government officials, including
-            the U.S. military and White House, to assist with response planning.
-          </Typography>
+          <BodyCopy source={content.introContent} linkTarget="_blank" />
           <SectionHeader variant="h4" component="h4" id="partners">
-            Partners
+            {content.partnersHeader}
           </SectionHeader>
-          <Typography variant="body1" component="p">
-            We work in partnership with the{' '}
-            <a
-              href="https://ghss.georgetown.edu/"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              Georgetown University Center for Global Health Science and
-              Security
-            </a>
-            ,{' '}
-            <a
-              href="http://med.stanford.edu/cerc.html"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              Stanford University Clinical Excellence Research Center
-            </a>
-            ,{' '}
-            <a
-              href="https://grandrounds.com/"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              Grand Rounds
-            </a>
-            , and{' '}
-            <a
-              href="https://globalhealth.harvard.edu/"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              Harvard Global Health Institute
-            </a>
-            .
-          </Typography>
-          <PartnerLogoGrid />
+
+          {content.partnersContent.map((section: any, i: number) => {
+            return (
+              <Fragment>
+                <BodyCopy source={section.copy} linkTarget="_blank" />
```

Comment:
do the links include `rel=noopener noreferrer`? If not we can try to figure this out later

Reply:
im going to punt this for now. im not seeing it as a prop in the react-markdown docs. gotta dig around a little more